common/utils/src/main/scala/org/apache/spark/util/SparkClassUtils.scala

```diff
@@ -50,6 +50,25 @@ private[spark] trait SparkClassUtils {
   def classIsLoadable(clazz: String): Boolean = {
     Try { classForName(clazz, initialize = false) }.isSuccess
   }
+
+  /**
+   * Determines whether the provided class is loadable in the current thread and a subclass
```

Comment:
`isAssignableFrom` is border than `a subclass of the given parent class`, isn't it?

Maybe, `classIsLoadableAndAssignableFrom` instead of `classIsLoadableAndSubOf`?

Reply:
Thank you @dongjoon-hyun, you are right.